### PR TITLE
[MIG][9.0] account multicurrency revaluation report

### PR DESCRIPTION
**Pull request by grindtildeath**
Depends on : https://github.com/OCA/account-closing/pull/44

### CODE REVIEW

File: account_multicurrency_revaluation_report/report/templates/unrealized_currency_gain_loss.xml [hunk 1]
```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (yvaucher) [inline account_multicurrency_revaluation_report/report/templates/unrealized_currency_gain_loss.xml]: To move in parent folder

**Review by yvaucher — COMMENTED** [review]
Please also delete mako template

File: account_multicurrency_revaluation/demo/account_demo.xml [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+    </record>
+    <record id="reval_journal" model="account.journal">
+        <field name="name">Revaluation Journal</field>
+        <field name="code">RJ</field>
+        <field name="type">general</field>
+    </record>
+    <record id="bank_journal_usd" model="account.journal">
+        <field name="name">Bank Journal USD</field>
+        <field name="code">BJ USD</field>
+        <field name="type">bank</field>
```
Comment (yvaucher) [inline account_multicurrency_revaluation/demo/account_demo.xml:68]: set currency_id in this file instead of doing it in unit tests ?

**Comment by grindtildeath** [comment]
Please remove work in progress label and add needs review.
Ping @fclementic2c @elicoidal

**Comment by fclementic2c** [comment]
ok for me
:+1:

**Pull request merged**